pygmt/helpers/__init__.py

```diff
@@ -4,6 +4,7 @@
 from pygmt.helpers.decorators import (
     deprecate_parameter,
     fmt_docstring,
+    insert_alias,
```

Comment:
Probably take this out of the top level import since it's not used anywhere else.

```suggestion
```